types/react-notification-system/react-notification-system-tests.ts

```diff
@@ -54,6 +53,10 @@ class MyComponent extends React.Component<any, any> {
             }
         };
 
-        return React.createElement(NotificationSystem, { title: "NotificationTitile", style: style, } as NotificationSystem.Attributes);
+        var ref = (instance: NotificationSystem.System) => {
+            this.notificationSystem = instance
+        }
+
+        return React.createElement(NotificationSystem, { title: "NotificationTitile", style: style, ref: ref } as NotificationSystem.Attributes);
```

Comment:
Good idea. I was not sure why test file was without it. I can't think of any good reason to have both `ts` and `tsx` files at once, so will transforming this test file to use `jsx` be sufficient?